SofaKernel/modules/SofaLoader/MeshVTKLoader.cpp

```diff
@@ -635,6 +667,34 @@ bool LegacyVTKReader::readFile(const char* filename)
                     }
 
                 }
+				else if (dataStructure == "NORMALS")
+				{
+					string dataName, dataType;
+					lnData >> dataName >> dataType;
+					inputNormals = newVTKDataIO(dataType, 3);
+					if (inputNormals == NULL) return false;
+					if (!inputNormals->read(inVTKFile, nb_ele, binary)) return false;
+				}
+				else if (dataStructure == "VECTORS")
+				{
+					string dataName, dataType;
+					lnData >> dataName >> dataType;
+					BaseVTKDataIO*  data = newVTKDataIO(dataType, 3);
+					if (data != NULL)
+					{
+						if (data->read(inVTKFile, nb_ele, binary))
+						{
+							inputDataVector.push_back(data);
+							data->name = dataName;
+							if (kw == "CELL_DATA"){
+								msg_info(this) << "Read cell data: " << data->name;
```

Comment:
Small tips: 
writing msg_info() is equivalent to msg_info(this) ...

Reply:
you are right we should do it for both or not at all. What do you prefer ?

Reply:
I prefer consistency :)

Reply:
both suggestions are consistent... well I'll do something